src/models/screen.js

```diff
@@ -0,0 +1,17 @@
+// const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const createScreenData = async (number) => {
+  const new_screen = await prisma.screen.create({
+    data: {
+      number,
+    },
+  });
+
+  // const movie_with_screen = getMovieDataById(new_screen.id);
+  return new_screen;
+};
+
+module.exports = {
+  createScreenData,
+};
```

Comment:
nice use of single responsibility thinking to extract the prisma code out into models!